src/main/resources/jenkins/plugins/slack/webhook/GlobalConfig/help-slackOutgoingWebhookURL.html

```diff
@@ -0,0 +1,5 @@
+<div>
+This is the webhook endpoint to send requests to.
```

Comment:
I think it would be good to point out here that leaving it blank will disable the functionality, and that the endpoint needs to be publicly accessible from the Slack servers. I think that will alleviate some concerns, and clarify why it isn't working when someone hasn't opened their firewall etc.